workspaces/marketplace/plugins/marketplace/src/components/MarketplacePluginInstallContent.tsx

```diff
@@ -199,67 +202,32 @@ export const MarketplacePluginInstallContent = ({
     codeEditor.setValue(yaml.stringify(dynamicPluginYaml));
   }, [codeEditor, packages]);
 
-  const showFullPlugin = React.useCallback(() => {
-    codeEditor.setValue(yaml.stringify(plugin));
-  }, [codeEditor, plugin]);
-
-  const [, copyToClipboard] = useCopyToClipboard();
-  const handleCopyToClipboard = React.useCallback(() => {
-    const value = codeEditor.getValue();
-    if (value) {
-      copyToClipboard(value);
-    }
-  }, [codeEditor, copyToClipboard]);
-
   const navigate = useNavigate();
-  const installationInstructions = plugin.spec?.installation;
   const examples = packages[0]?.spec?.appConfigExamples;
-  const showRightCard = installationInstructions || examples;
+  const installationInstructions = plugin.spec?.installation;
+  const aboutMarkdown = plugin.spec?.description;
+  const availableTabs = [
+    examples && {
+      label: 'Examples',
+      content: examples,
+      key: 'examples',
+      others: { packageName: packages[0].spec?.dynamicArtifact },
+    },
+    installationInstructions && {
+      label: 'Setting up the plugin',
+      content: installationInstructions,
+      key: 'installation',
+    },
+    aboutMarkdown && {
+      label: 'About the plugin',
+      content: aboutMarkdown,
+      key: 'about',
+    },
+  ].filter(tab => tab !== undefined) as TabItem[];
```

Comment:
when I set `installationInstructions = ''`
The UI looks like
![Screenshot 2025-03-20 at 3 32 53 PM](https://github.com/user-attachments/assets/5670c1c0-fd6c-4820-855d-581186ca82f6)

This should work

```suggestion
  ].filter(tab => tab) as TabItem[];
```